PixiEditor/PixiEditor.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk.WindowsDesktop">
 
   <PropertyGroup>
-    <OutputType>WinExe</OutputType>
+    <OutputType>Exe</OutputType>
```

Comment:
Hi, thanks for PR! I have one question, why exactly you changed output type from `WinExe` to `Exe`?

Reply:
Hey, sorry about that I changed the ouput to a Console Application for testing purposes and forgot to change it back to a Windows Application.

Reply:
Could you change that back? Thanks